spec/lib/redmine/scm/base_spec.rb

```diff
@@ -39,12 +39,12 @@
 
     before do
       Repository.const_set('TestScm', test_scm_class)
-      Redmine::Scm::Base.add 'TestScm'
+      OpenProject::Scm::Manager.add 'TestScm'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.